AmplifyPlugins/Logging/Sources/AWSCloudWatchLoggingPlugin/AWSCloudWatchLoggingSessionController.swift

```diff
@@ -164,7 +164,22 @@ final class AWSCloudWatchLoggingSessionController {
     }
     
     func flushLogs() async throws {
-        try await session?.logger.flushLogs()
+        guard let logBatches = try await session?.logger.getLogBatches() else { return }
+                
+        for batch in logBatches {
+            try await consumeLogBatch(batch)
```

Comment:
Just curious, what will happen here if first item in `logBatches` will throw an error? Will rest of the items be cleared?

Reply:
no, just the item that fails.